examples/example-xds/README.md

```diff
@@ -24,8 +24,8 @@ This creates the scripts `build/install/example-xds/bin/hello-world-client-xds`
 ### Run the example without using XDS Credentials
 
 To use XDS, you should first deploy the XDS management server in your deployment environment
-and know its name. You need to set the `GRPC_XDS_BOOTSTRAP` environment variable to point to the
-gRPC XDS bootstrap file (see
+and know its name. You need to set the `GRPC_XDS_BOOTSTRAP` environment variable or the `io.grpc.xds.bootstrap` java
```

Comment:
should we say something like below to be more precise?:

...`GRPC_XDS_BOOTSTRAP` environment variable (preferred) or if that is not set then the `io.grpc.xds.bootstrap` java system property...

Reply:
Makes sense!
I'll update it :slightly_smiling_face: